openedx/core/djangoapps/catalog/utils.py

```diff
@@ -119,6 +119,30 @@ def get_program_types(name=None):
         return []
 
 
+def get_currency():
+    """Retrieve currency data from the catalog service.
+
+    Returns:
+        list of dict, representing program types.
+        dict, if a specific program type is requested.
+    """
+    catalog_integration = CatalogIntegration.current()
+    if catalog_integration.enabled:
+        try:
+            user = catalog_integration.get_service_user()
+        except ObjectDoesNotExist:
+            return []
+
+        api = create_catalog_api_client(user)
+        cache_key = '{base}.currency'.format(base=catalog_integration.CACHE_KEY)
+
+        data = get_edx_api_data(catalog_integration, 'currency', api=api,
```

Comment:
Nit: also just return `get_edx_api_data` instead of the `data` variable assignement?

Reply:
changed